examples/toolbar/toolbar.html

```diff
@@ -44,32 +51,123 @@ <h1>Toolbar Example</h1>
     <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="ex1">
-        <div class="toolbar" role="toolbar">
-            <div class="toolbar-item selected" role="button" tabindex="0">
-                Tool 1
+      <div class="format" role="toolbar" aria-label="Text Formatting" aria-controls="textarea1">
```

Comment:
The way that most screen readers render toolbars, I don't know how it be possible for them to exploit aria-controls on a toolbar container. I think we should not have it here.

Reply:
Actually, on further investigation, at least JAWS is seeing aria-controls on each button, and it enables movement to the text area from the button if in eading mode. I didn't expect that specifying aria-controls on the parent container would do that -- very interesting. So, at least there is some support for aria-controls in this scenario. The question is whether we want this to be part of the pattern.